.github/workflows/system-tests.yml

```diff
@@ -68,12 +65,13 @@ jobs:
       LIBRARY_INJECTION_CONNECTION: ${{ matrix.lib-injection-connection }}
       LIBRARY_INJECTION_ADMISSION_CONTROLLER: ${{ matrix.lib-injection-admission-controller }}
       LIBRARY_INJECTION_INIT_IMAGE: ghcr.io/datadog/dd-trace-py/dd-lib-python-init:${{ github.sha }}
-      LIBRARY_INJECTION_TEST_APP_IMAGE: ghcr.io/datadog/dd-trace-py/dd-lib-python-init-test-app:${{ github.sha }}
+      LIBRARY_INJECTION_TEST_APP_IMAGE: ghcr.io/datadog/dd-trace-py/${{ matrix.lib-injection-test-app }}:${{ github.sha }}
     steps:
       - name: Checkout system tests
         uses: actions/checkout@v3
         with:
           repository: 'DataDog/system-tests'
+          ref: kylev/fix-lib-injection
```

Comment:
Is the plan to merge this branch before merging this PR?

Reply:
we can do before or after, this PR is not dependent on https://github.com/DataDog/system-tests/pull/581